bundler.d/nulldb.rb

```diff
@@ -1 +1,3 @@
-gem 'activerecord-nulldb-adapter'
+group :nulldb do
+  gem 'activerecord-nulldb-adapter'
```

Comment:
Untested, but does this work?
```suggestion
  gem 'activerecord-nulldb-adapter', require: 'nulldb/rails'
```

Reply:
no